### ISSUE TITLE
Commit causes crash: The 'build-jobs' config option should be an integer

### ISSUE THREAD

**Issue by jberry-suse** [openSUSE/osc#372]
1a92c8b995f762f9357fa9630d7981ab1ab3497c

```
Traceback (most recent call last):
  File "./osc-wrapper.py", line 41, in <module>
    r = babysitter.run(osccli)
  File "/home/jberry/osc/osc/babysitter.py", line 61, in run
    return prg.main(argv)
  File "/home/jberry/osc/osc/cmdln.py", line 336, in main
    self.postoptparse()
  File "/home/jberry/osc/osc/commandline.py", line 138, in postoptparse
    override_verbose = self.options.verbose)
  File "/home/jberry/osc/osc/conf.py", line 856, in get_config
    config[opt] = meth('general', opt)
  File "/usr/lib64/python2.7/ConfigParser.py", line 359, in getint
    return self._get(section, int, option)
  File "/usr/lib64/python2.7/ConfigParser.py", line 356, in _get
    return conv(self.get(section, option))
  File "/usr/lib64/python2.7/ConfigParser.py", line 623, in get
    return self._interpolate(section, option, value, d)
  File "/usr/lib64/python2.7/ConfigParser.py", line 691, in _interpolate
    self._interpolate_some(option, L, rawval, section, vars, 1)
  File "/usr/lib64/python2.7/ConfigParser.py", line 700, in _interpolate_some
    p = rest.find("%")
AttributeError: 'int' object has no attribute 'find
```

Blocks our CI as well https://travis-ci.org/openSUSE/osc-plugin-factory/jobs/332665417.

**Comment by jberry-suse** [comment]
Would likely be a good idea to add a test that just invoked `osc` directly to pick up any initialization errors.

**Comment by marcus-h** [comment]
On 2018-01-24 07:48:59 +0000, Jimmy Berry wrote:
> Traceback (most recent call last):

<SNIP>

>   File "/usr/lib64/python2.7/ConfigParser.py", line 700, in _interpolate_some
>     p = rest.find("%")
> AttributeError: 'int' object has no attribute 'find
> 
> Blocks our CI as well https://travis-ci.org/openSUSE/osc-plugin-factory/jobs/332665417.
> 
Oops, fixed in commit d0213c6.
